### PR TITLE
Admin

### PR DESCRIPTION
**Pull request by Praj0810**
Admin SignUp schema And Api Added

### CODE REVIEW

File: Controller/admin_login_signup.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+const Admin = require("../Model/AdminSchema");
+const bcrypt = require("bcrypt");
+const jwt = require('jsonwebtoken');
+const jwtSecret = "qwerty131";
```
Comment (bidyut-neo) [inline Controller/admin_login_signup.js]: please move `qwerty131`  to a .env file

File: Controller/admin_login_signup.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  
+}
+
+
+
+const login = async (req, res) => {
+    try {
+      const { EmailId, Password } = req.body;
+      if (!EmailId || !Password) {
+        return res.status(400).send({ error: "Plz enter the data" });
```
Comment (bidyut-neo) [inline Controller/admin_login_signup.js]: `Plz ` should be `Please`

File: Middleware/AuthJWT.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+const jwt = require('jsonwebtoken');
+const jwtSecret = "qwerty131";
```
Comment (bidyut-neo) [inline Middleware/AuthJWT.js:2]: `qwerty131 ` should be moved to .env file

File: Model/AdminSchema.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+const mongoose = require("mongoose");
+
+const AdminLoginSchema = new mongoose.Schema({
+    Admin_Account_Address : {
+        type: String,
+        require: true
+    },
+    First_Name:{
```
Comment (bidyut-neo) [inline Model/AdminSchema.js:8]: `First_Name` this entire case is confusing. It should be either `firstName`, `first_name`, `FirstName` depending on the programming language. `_` and capital letters are not used together. Please fix everywhere.

File: Routes/adminRoute.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const Router = express.Router();
+const { signUp, login }= require('../Controller/admin_login_signup');
```
Comment (bidyut-neo) [inline Routes/adminRoute.js]: In JS we follow `camelCase`. The file names do not contain `_`. it should be `adminLoginSignup`. This should be reflected everywhere. e.g. in Routes, `manage_kyc_details` should become `manageKycDetails`

**Review by bidyut-neo — CHANGES_REQUESTED** [review]

File: Controller/admin_login_signup.js (hunk 1 above)
Comment (Praj0810) [reply Controller/admin_login_signup.js]: Moved to .env

File: Routes/adminRoute.js (hunk 5 above)
Comment (Praj0810) [reply Routes/adminRoute.js]: Did the camelCase change in the file Names

File: Controller/admin_login_signup.js (hunk 2 above)
Comment (Praj0810) [reply Controller/admin_login_signup.js]: Did the change with Please instead on Plz

**Review by bidyut-neo — APPROVED** [review]

**Pull request merged**